LICENSE

```diff
@@ -1,21 +1,21 @@
-    MIT License
+MIT License
 
-    Copyright (c) Microsoft Corporation. All rights reserved.
+Copyright (c) 2019 Guilherme Lucas 

```

Comment:
I'm guessing Microsoft is going to have a problem with this changing of the license but idk